api/src/models/settings.js

```diff
@@ -0,0 +1,48 @@
+const db = require('../db/connection')
+const { cryptr } = require('../config')
+const { identity } = require('ramda')
+
+let encrypt = cryptr.encrypt.bind(cryptr)
+let decrypt = cryptr.decrypt.bind(cryptr)
+
+// When testing, do not encrypt or decrypt settings
+//
+if (process.env.NODE_ENV === 'test') {
+  encrypt = decrypt = identity
```

Comment:
@kamicut I have been running in my dev env: `time yarn test` with this code block present and also commented out, and it does not seem to offer any performance boost to testing. It always takes about "real  1m9.549s" approximate length of time. Consider removing this conditional to simplify the code in settings.js?

As I think you mentioned on the call: this would also mean the settings.js model is tightly coupled to the Cryptr class. But the model is also coupled with the database connection so settings.test.js is already more of an integration test than a unit test. As a separate effort we could look at dependency injection as an alternative?

Reply:
@guidorice I like your suggestion for decoupling, but I feel that it might be a larger refactor since our models are all coupled to the database. We could take a look at our tests in another larger issue maybe?

Reply:
This sounds good to me.